packages/block-library/src/navigation/index.php

```diff
@@ -344,18 +344,22 @@ function block_core_navigation_maybe_use_classic_menu_fallback() {
 		return;
 	}
 
+
 	// Create a new navigation menu from the classic menu.
 	$wp_insert_post_result = wp_insert_post(
 		array(
 			'post_content' => $classic_nav_menu_blocks,
-			'post_title'   => $classic_nav_menu->slug,
+			'post_title'   => $classic_nav_menu->name,
```

Comment:
This was a hidden bug caught by the new tests I added.

Reply:
I'm going to pull this out into a separate PR.

Reply:
Here is the cherry picked PR

https://github.com/WordPress/gutenberg/pull/48771

I'll rebase here once that is ✅

Reply:
✅